.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,19 +18,23 @@ What does this PR do?
 
 ## Testing Instructions
 
-How can the Archive's QA team verify that this is working as you intended? (If you have access, please copy this into the JIRA ticket for them!)
+How can the Archive's QA team verify that this is working as you intended?
+
+(If you have a JIRA account with access, please update or comment on the issue
+with any new or missing testing instructions.)
 
 ## References
 
-Are there any other relevant issues / PRs / mailing lists discussions related to this?
+Are there other relevant issues / pull requests / mailing lists discussions?
```

Comment:
Can we get rid of the spaces around the slashes? Also, it should probably be "mailing list discussions" -- no `s` on lists.